app/src/main/java/co/touchlab/kampstarter/android/MainActivity.kt

```diff
@@ -26,7 +26,7 @@ class MainActivity : AppCompatActivity() {
             Snackbar.make(breed_list, errorMessage, Snackbar.LENGTH_SHORT).show()
         })
 
-        adapter = MainAdapter(model::updateBreedFavorite)
+        adapter = MainAdapter { model.updateBreedFavorite(it) }
```

Comment:
Is this just for readability or is there a functional difference here I'm missing?

Reply:
Since the function now returns a Job, it doesn't match the signature the adapter is expecting (returning Unit). It felt more correct to change this then to introduce Job in the adapter which doesn't actually care about it